cla-backend-go/v2/cla_manager/handlers.go

```diff
@@ -108,19 +109,18 @@ func Configure(api *operations.EasyclaAPI, service Service, LfxPortalURL string,
 	api.ClaManagerInviteCompanyAdminHandler = cla_manager.InviteCompanyAdminHandlerFunc(func(params cla_manager.InviteCompanyAdminParams) middleware.Responder {
 
 		// Get Contributor details
-		usc := v2UserService.GetClient()
-		user, userErr := usc.GetUserByUsername(params.UserLFID)
+		user, userErr := easyCLAUserRepo.GetUser(params.UserID)
 
 		if userErr != nil {
-			msg := fmt.Sprintf("Problem getting user by LFID : %s, error: %+v ", params.UserLFID, userErr)
+			msg := fmt.Sprintf("Problem getting user by ID : %s, error: %+v ", params.UserID, userErr)
 			return cla_manager.NewInviteCompanyAdminBadRequest().WithPayload(
 				&models.ErrorResponse{
 					Code:    "400",
 					Message: msg,
 				})
 		}
 
-		claManagerDesignee, err := service.InviteCompanyAdmin(params.Body.ContactAdmin, params.Body.CompanySFID, params.Body.ProjectSFID, params.Body.UserEmail, *user.Emails[0].EmailAddress, user.Name, LfxPortalURL)
+		claManagerDesignee, err := service.InviteCompanyAdmin(params.Body.ContactAdmin, params.Body.CompanySFID, params.Body.ProjectSFID, params.Body.UserEmail, user.UserGithubUsername, user.UserGithubID, LfxPortalURL)
```

Comment:
Consider passing the User object - such that the serivce/impl can pull what it needs out of the user object based on GH or Gerrit.